app.js

```diff
@@ -13,6 +13,7 @@ var multer = require('multer');
 var upload = multer({
   limits: { fileSize: 1024*1024 }
 });
+var upload = multer();
```

Comment:
#### Automated PR Comment From coverity
<!-- b0f6e0bb3057d04429acf3a6b6c780f3 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - File upload with insecure storage
_Low CWE-921_
The `multer` middleware is configured to use memory for storing file uploads rather than persisting them on disk.
It is possible to cause a denial of service attack by uploading a large number of files or one big file into the server's memory.
### How to fix
Avoid using `MemoryStorage` in production environments. Configure `multer` to store file uploads on a disk by setting the `storage` option to the built-in `diskStorage` module or configuring the `dest` option as a folder on disk where the files are stored.